firebase/rules/tests/docker-images.spec.ts

```diff
@@ -0,0 +1,33 @@
+import 'jest';
+import * as matchers from '../matchers';
+import * as targaryen from 'targaryen';
+
+import { SYSTEM_USER } from '@machinelabs/models';
+
+const anonymousUser = targaryen.util.users.anonymous;
+const rules = require('../database.rules.json');
+
+describe('/docker_images', () => {
+  const currentUser = { uid: '1', email: 'foo@bar.com', provider: 'github' };
+  const systemUser = { uid: SYSTEM_USER };
+
+  beforeEach(() => {
+    jest.addMatchers(matchers);
+
+    targaryen.util.setFirebaseData();
+
+    targaryen.util.setFirebaseRules(rules);
+  });
+
+  it('All users can read into common docker images', () => {
+    expect(anonymousUser).canRead('/docker_images/common');
+    expect(currentUser).canRead('/docker_images/common');
+    expect(systemUser).canRead('/docker_images/common');
+  });
```

Comment:
Doesn't hurt to have a test like this, but I think it doesn't add a ton of value. If the gates are open to everyone anyways, there's no need to be concerned about people accessing it. Also, if we'd introduce other type of users, this test would need to be updated as well.